apps/condo/domains/user/integration/appleid/AppleIdIdentityIntegration.js

```diff
@@ -0,0 +1,154 @@
+// eslint-disable-next-line import/order
+const axios = require('axios').default
+
+// eslint-disable-next-line import/order
+const dayjs = require('dayjs')
+const jose = require('jose')
```

Comment:
This package is not listed in `package.json`.

Reply:
10x I'll add it into dependencies